analyzers/haskell/analyzer/stack.go

```diff
@@ -0,0 +1,70 @@
+package analyzer
+
+import (
+	"github.com/fossas/fossa-cli/buildtools/stack"
+	"github.com/fossas/fossa-cli/graph"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+func AnalyzeStack(m module.Module) (graph.Deps, error) {
+	stackAllDeps, err := stack.GetStackDependencies(m.Dir, nil)
```

Comment:
I need to set up a Haskell environment to try this out, but my initial question is, is it really necessary to shell out to `stack` here three times? I trust your judgment so I won't block on this, but I am curious to know if there is a simpler way in the future.

Reply:
There's a feature landing soon in stack that'll output the dependency graph as a proper tree; in the meantime the commands available don't give the context we need.

`stack ls dependencies (--depth N)` outputs a raw list of dependencies without any information about relationships
`ghc-pkg dot` gives us the information about relationships between packages